tasks/lzards-backup/src/index.ts

```diff
@@ -288,7 +323,9 @@ export const backupGranule = async (params: {
       granuleId: granule.granuleId,
     })));
   } catch (error) {
-    if (error.name === 'CollectionNotDefinedError') {
+    if (error.name === 'CollectionIdentifiersNotProvidedError') {
```

Comment:
Sorry, didn't think of this before but I think we might want to do:

```
if (error instanceof CollectionIdentifiersNotProvidedError)
```
rather than a comparison to the error.name string.

Reply:
https://github.com/nasa/cumulus/pull/3043/commits/e55240184f9a102ea07e60e8fdd5559546d1705c#diff-31ab5953f1d25d2a7e338fb25acc5c7ae53aacc8cde330d0084eb4e00236cf3fR322-R324 Good point!